### ISSUE TITLE
Can you provide you hyperparameter settings of DETReg in Few-Shot Object Detection?

### ISSUE THREAD

**Issue by miznchimaki** [amirbar/DETReg#51]
Thanks for your great work very much!
I'm intersted in using your unsupervised pretrained DETReg in few shot object detection, so can you provide the hyperparameter of DETReg in FSOD? (such as total epochs/iterations, optimizer, initial learning rate, learning rate dropping iterations/epochs, weight decay and so on). Or maybe your hyperparameter of FSOD is same as Meta-DETR? (becaused I noticed you compared your results with Meta-DETR.)
Sincerely waiting for your reply!

**Comment by amirbar** [comment]
Hello, apologies for dropping the ball here. However, we've found an issue in our evaluation pipeline and I'm now uploading a new version with a fix. The few-shot checkpoints are now made available including some instructions to reproduce in the readme. These are the up-to-date few-shot results we are seeing:

evaluation type | Novel AP 10 | Novel AP 30 | Novel AP75 10 | Novel AP75 30
-- | -- | -- | -- | --
Supervised | 12.4 | 20.4 | 13.3 | 21.8
DETReg | 13.7 | 22.6 | 15.1 | 24.3
